### PR TITLE
add `state cve` command

### PR DESCRIPTION
**Pull request by mdrohmann**
https://www.pivotaltracker.com/story/show/176449951

### CODE REVIEW

**Comment by mdrohmann** [comment]
I don't know if this ready for prime time due to some limitations of the mediator that came up:

Follow-up stories are here: 
- https://www.pivotaltracker.com/story/show/176519572
- https://www.pivotaltracker.com/story/show/176508772
- https://www.pivotaltracker.com/story/show/176508740

The first one might be a blocker for putting it on production...

File: cmd/state/internal/cmdtree/cve.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	"github.com/ActiveState/cli/internal/runners/cve"
+)
+
+func newCveCommand(prime *primer.Values) *captain.Command {
+	runner := cve.NewCve(prime)
+
+	cmd := captain.NewCommand(
+		"cve",
+		locale.Tl("cve_title", "CVE Summary"),
+		locale.Tl("cve_description", "Print project vulnerabilities"),
```
Comment (Naatan) [inline cmd/state/internal/cmdtree/cve.go:16]:
```suggestion
		locale.Tl("cve_description", "Show a summary of CVE vulnerabilities"),
```

**Review by Naatan — CHANGES_REQUESTED** [review]
Very nice work! Just one super nitpicky change.

**Comment by mdrohmann** [comment]
I added the changes that were necessary to work with the updated mediator.  Of course, this does not work yet, with the production endpoint for the mediator.

**Review by Naatan — DISMISSED** [review]

**Comment by mdrohmann** [comment]
Took a little while until the mediator was deployed, because kubernetes ran out of some resources, but now everything seems to be working.  Just need another approval for the latest additions that I had to do...

**Review by Naatan — APPROVED** [review]

**Pull request merged**